docs/README.md

```diff
@@ -87,6 +88,30 @@ For example:
 },
 ```
 
+The `product-price-suspense` block, in turn, does not receive any props - when declared, it automatically renders a loading spinner. However, you can also pass to it children blocks you want to render instead. For example:
```

Comment:
It should be possible to use pass a Fallback slot prop to change the spinner to something else, just like the React suspense API: https://pt-br.reactjs.org/docs/concurrent-mode-suspense.html

Reply:
Done! Now we have the `Fallback` prop